libtiledbvcf/CMakeLists.txt

```diff
@@ -80,20 +80,40 @@ endif()
 
 # Add some global compiler flags
 string( TOUPPER "${CMAKE_BUILD_TYPE}" BUILD_TYPE)
-if (BUILD_TYPE STREQUAL "DEBUG")
-  add_compile_options(-DDEBUG -O0 -g3 -ggdb3 -gdwarf-3 -Wall)
-elseif (BUILD_TYPE STREQUAL "RELEASE")
-  add_compile_options(-DNDEBUG -O3 -Wall -Werror)
+if(WIN32 AND NOT MSYS)
+  #Note: seems this branch is taken even when -G "MSYS Makefiles" was used to invoke at top-level!
+  #(cmake --version was 3.24.1 when this apparent failure encountered...)
+  message(STATUS "configuring for WIN32, MSYS is \"${MSYS}\", MSVC is \"${MSVC}\"")
+  #Note: also seems MSVC not yet set at this point, so can't check 'if (MSVC)'.
+  #TBD: figure out approp. compile options to set for WIN32/MSVC if they need to
+  # be different from cmake defaults
+  if (BUILD_TYPE MATCHES "[DD][EE][BB][UU][GG]")
+    set(CMAKE_BUILD_TYPE "Debug")
+  elseif (BUILD_TYPE MATCHES [Rr][Ee][Ll][Ww][Ii][Tt][Hh][Ii][Nn][Ff][Oo])
+    set(CMAKE_BUILD_TYPE "RelWithDebInfo")
+  elseif (BUILD_TYPE MATCHES "[Rr][Ee][Ll][Ee][Aa][Ss][Ee]")
+    set(CMAKE_BUILD_TYPE "Release")
+  endif()
+  message(WARNING "Building without deprecation warnings")
+  # /wd4996 - no deprecated
+  add_compile_options(/wd4996)
+else()
+  if (BUILD_TYPE STREQUAL "DEBUG")
```

Comment:
Please remove all of this stuff, cmake build types are CamelCase in documentation (possibly case-insensitive internally, but that's not our concern).

Reply:
This is not mine, that was pre-existing, tiledb-vcf team (?) was apparently generally using upper-case converting somewhere else.  Couple lines below that is also a pre-existing "RELEASE".